<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicWollastonLiftForm</class>
 <widget class="QWidget" name="BasicWollastonLiftForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>329</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Wollaston</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Wollaston Lift</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="StatusConnected">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>not connected</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="StatusCmode">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>NONE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelCmode">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Control:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonTakeLocal">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>31</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Take Local Control</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxOverride">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>63</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>override</string>
   </property>
  </widget>
  <widget class="QFrame" name="cModeFrame">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>5</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Current Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusStatus">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>unkown</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>In</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Out</string>
   </property>
  </widget>
  <zorder>cModeFrame</zorder>
  <zorder>title</zorder>
  <zorder>StatusConnected</zorder>
  <zorder>StatusCmode</zorder>
  <zorder>LabelCmode</zorder>
  <zorder>ButtonTakeLocal</zorder>
  <zorder>checkBoxOverride</zorder>
  <zorder>label_7</zorder>
  <zorder>statusStatus</zorder>
  <zorder>upButton</zorder>
  <zorder>downButton</zorder>
 </widget>
 <tabstops>
  <tabstop>checkBoxOverride</tabstop>
  <tabstop>ButtonTakeLocal</tabstop>
 </tabstops>
 <resources>
  <include location="../visao.qrc"/>
 </resources>
 <connections/>
</ui>
